### ISSUE TITLE
riot-web download audio not working

### ISSUE THREAD

**Issue by metasintactic** [vector-im/riot-web#6280]
### Description

The riot-web desktop client on linux will not save an audio file. The audio file (in this case a .wav < 2MB) shows up in the chat with a control bar, can be played, etc. When the download file button is clicked, a save file dialogue pop's up with the word "download" in the filename field. Selecting a download location and clicking "Save" with or without changing the filename field, simply dismisses the dialogue without actually downloading the audio file.

I don't know what other platforms or media types may be affected.

### Steps to reproduce

- Upload a short audio file to a chat.
- Click the download button on the audio file's control bar.
- Save the file.
- Try to find the file on the hard disk.

### Version information

Riot Desktop on Linux
riot-web version: 0.13.5
olm version: 2.2.1
Homeserver is https://matrix.org
Identity Server is https://vector.im

**Comment by lampholder** [comment]
I wasn't able to verify this with an 8.8 MB wav file on Riot 0.13.5 on Ubuntu - I ran out of patience trying to source a bigger wav so catted the 8.8 MB one together three times, but that also didn't trigger the same failure mode.

Is there anything interesting about the chat? Is it E2E encrypted?

**Comment by lampholder** [comment]
Ah yes, I _can_ reproduce it when the chat is encrypted. Looks like the default browser rendering of a wav file doesn't play nicely with the restrictions of E2E encryption.

**Comment by lampholder** [comment]
It's actually worse via a real browser (not electron) - you just fail to download a file called 'download'.

Clicking the download link beneath the media control works fine.

**Comment by metasintactic** [comment]
Yes. I think you already figured that out, but e2e was turned on.
